### ISSUE TITLE
[Bug] Arcane Barrier pushing out owner after respawn

### ISSUE THREAD

**Issue by sans01hp** [WinDanesz/AncientSpellcraft#118]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Observed behaviour

Ritual of Arcane Barrier pushs player out as if it was a hostile entity after his death.

I like to put my spawnpoint inside the Arcane Barrier for extra protection, but after respawning, the barrier push me up making me stuck on the barrier itself and it doesn't let me enter after that.

This  stops only after  going out of the render distance and coming back. But it's kinda impossible with the respawn being inside the barrier... unless you have a way to get unstuck of it.

### Expected behaviour

Arcane barrier only pushing hostile entities or non ally players.

### Steps to reproduce

1.Join Singleplayer
2.Setup and activate the Ritual of Arcane Barrier inside your respawn chunk.
3.use /kill or just get killed.

### Environment (Singleplayer/Server, etc.)

Singleplayer

### Mod version

Ancient Spellcraft 1.5.5

### Forge version

1.12.2- 14.23.5.2860

### Other mods

Electroblob's Wizardry 4.3.8

**Comment by dil-dvinnai** [comment]
By saying that the barrier push you up, do you literally mean that you are stuck on the top of the barrier?
Moving out of the render distance always allows you to return?

**Comment by WinDanesz** [comment]
By saying that the barrier push you up, do you literally mean that you are stuck on the top of the barrier?
Moving out of the render distance always allows you to return?

**Comment by sans01hp** [comment]
Yes. Even on creative i get stuck on the barrier. It tries to push you up if you're not close from the border. Without flying, you keep constantly falling and getting pushed up by the barrier, causing the stuck. Only after getting unstuck and unloading the chunk by going out the render distance, it resets the barrier to consider you an ally again.

**Comment by sans01hp** [comment]
Also, i was planning doing another issue i found but i don't know if it was intended.  Runic plates spawning on battle keeps spawns as [Empty Slot]. Is that a bug or do you plan only being able to put spells on the spell blade via  runic plates?  [First putting the spell on the plate and then the plate on the blade.]

**Comment by sans01hp** [comment]
Yeah... That's another bug. You can't  put spells on these empty plates. Only runic plates droped by evil class Wizards are matching the ones in creative tab.

**Comment by WinDanesz** [comment]
Empty plates shouldnt exist. Simply there are no spells for novice tier yet. Adding some will solve it

**Comment by sans01hp** [comment]
About that... I Just don't know if i'm unlucky... but all plates i've found from structures are spawning like that. "Novice, empty". Only plates dropped from wizards are above novice tier. Adding novice tier spells solves the "empty"problem, but i don't know if that's causing some tier conflict loot spawning in structures. maybe i'm just unlucky, but that happening more than 3 times (when the plate actually spawn, and spawn as a novice tier) it's unlikely to be a coincidence.
